regression/pages/common/api_clients.py

```diff
@@ -0,0 +1,260 @@
+"""
+API classes for different tasks
+"""
+import Cookie
+import json
+
+import requests
+from requests.auth import AuthBase
+from edx_rest_api_client.client import EdxRestApiClient
+
+from regression.pages.whitelabel.const import (
+    ACCESS_TOKEN,
+    AUTH_PASSWORD,
+    AUTH_USERNAME,
+    ECOMMERCE_API_URL,
+    ENROLLMENT_API_URL,
+    URL_WITHOUT_AUTH
+)
+
+
+class ApiException(Exception):
+    """
+    Exceptions for API failures
+    """
+    pass
+
+
+def check_response(response):
+    """
+    Check whether a response was successful. If not, raise an exception
+    Args:
+        response:
+    """
+    if response.status_code != 200:
+        raise ApiException(
```

Comment:
In the case of a non-200 code though, I would expect there to be less text. My main point is to think through what will help someone who is troubleshooting the issue. If you think the text is going to be more noise than it's worth, then that's ok. You can stick with what you have. But pls look at a couple of examples of failures before you decide.